frontend/views/containers/proposals/Mincome.vue

```diff
@@ -1,33 +1,38 @@
 <template lang='pug'>
   proposal-template(
+    ref='proposal'
     :title='L("Change minimum income")'
     :rule='{ value: 8, total: 10 }'
-    :disabled='$v.form.$invalid || ($v.steps[config.steps[ephemeral.currentStep]] && $v.steps[config.steps[ephemeral.currentStep]].$invalid)'
+    :disabled='$v.form.$invalid'
     :maxSteps='config.steps.length'
     :currentStep.sync='ephemeral.currentStep'
     @submit='submit'
   )
 
     label.field(v-if='ephemeral.currentStep === 0' key='0')
       i18n.label New minimum income
-      .input-combo
+      // REVIEW: why $error is never true, even if we type 0?
```

Comment:
@pieer I need your help here ☝️

Reply:
I think I stumbled upon this problem this week.. I fixed this with that line: minValue: (val) => val > 0, but I guess it's the same as "minValue (value) { return value > 0 }" right?

Reply:
Is required and decimal working?

Reply:
Oh, I found the bug! It was missing `$model`
```diff
- v-model='$v.form.incomeProvided'
+ v-model='$v.form.incomeProvided.$model'
```